site2/website-next/sidebars.json

```diff
@@ -324,7 +324,25 @@
       "items": [
         "reference-terminology",
         "reference-cli-tools",
-        "reference-configuration",
+        {
+          "type": "category",
+          "label": "Pulsar Configuration",
+          "link": {
+            "type": "doc",
+            "id": "reference-configuration"
+          },
+          "items": [
+            "reference-configuration-bookkeeper",
+            "reference-configuration-broker",
+            "reference-configuration-client",
+            "reference-configuration-log4j",
+            "reference-configuration-log4j-shell",
+            "reference-configuration-standalone",
+            "reference-configuration-websocket",
+            "reference-configuration-pulsar-proxy",
+            "reference-configuration-zookeeper"
+          ]
+        },
```

Comment:
changes from sidebars.json also need to be submitted in repo `apache/pulsar` but not here. @SignorMercurio /cc @Anonymitaet